### ISSUE TITLE
Provide mechanism to break long lines in Style

### ISSUE THREAD

**Issue by keenancrane** [penrose/penrose#1002]
**Is your feature request related to a problem? Please describe.**
Currently a long line of Style code can't be continued after a newline.  Breaking lines into multiple lines is helpful for increasing legibility, especially in gnarly graphics code.

An example (which is not particularly long…) is something like writing
```
   ensure greaterThan( angleFrom(pj-pi,pl-pi), toRadians(30.5) )
```
as
```
   ensure greaterThan(
      angleFrom(pj-pi,pl-pi),
      toRadians(30.5)
   )
```

**Describe the solution you'd like**
It's probably too aggressive (especially in the current iteration of Style) to allow arbitrary newlines.  A good solution for now would be to just use a special character to indicate a break.  I propose we use the backslash `\`, which is common in other languages, and currently unused in Style.  The example above then becomes
```
   ensure greaterThan( \
      angleFrom(pj-pi,pl-pi), \
      toRadians(30.5) \
   )
```
Not beautiful, but it gets the job done.

**Describe alternatives you've considered**
One alternative is to, e.g., define temporary variables in order to get shorter expressions.  Sometimes this is the right way, but other times it adds a bunch of dummy variables to the program that are never referenced again, and only make the code harder to read.

**Comment by liangyiliang** [comment]
In C/C++ we use semicolon to denote a statement, regardless of newline characters. Is that something we can do here? i.e. End every _statement_ with a semicolon, and allow newlines within statements ...

**Comment by samestep** [comment]
@liangyiliang Yeah I like that approach. Not sure how many others on the team agree, though...
